script.js

```diff
@@ -39,4 +36,175 @@ window.onresize = () => {
   }
 };
 
+// Model
+const overlayModal = document.getElementById('overlayModal');
+function openModal(modal) {
+  if (modal == null) return;
+  modal.classList.add('active');
+  window.scrollTo(0, 0);
+  overlayModal.classList.add('active');
+  navLinks.forEach((link) => {
+    link.style.color = '#c1c7d0';
+  });
+}
+
+function closeModal(modal) {
+  if (modal == null) return;
+  modal.classList.remove('active');
+  modal.innerHTML = '';
+
+  overlayModal.classList.remove('active');
+  navLinks.forEach((link) => {
+    if (w > 768) {
+      link.style.color = '#344563';
+    } else {
+      link.style.color = '#fff';
+    }
+  });
+}
+
+function createCard(project, index) {
+  const dataStr = encodeURIComponent(JSON.stringify(project));
+  const projectTemplate = `<div class="card">
+        <header id="header_${(index += 1)}" >
+          <img class="portfolio" src="./images/${project.featureImage}" 
+          alt="Portfolio">
+         </header>
+         <div class="card_body cb_${index}">
+          <h2 class="title">${project.name}</h2>
+          <div class="experience">
+          <span class="subtitle">Canopy</span>
+          <ul class="block">
+            <li>Back End Dev</li>
+            <li>2015</li>
+          </ul>
+          </div>
+          <p class="content">${project.description}
+          </p>              
+          <ul class="skills project_${index}">
+          </ul>
+          <button class="button" data-project=${dataStr} data-toggle="modal" data-modal-target="#modal" > See project </button>
+         </div>
+        </div>`;
```

Comment:
- [ ] Please use the correct color model buttons text. `#6070FF1

### Your design

![image](https://user-images.githubusercontent.com/83803414/217812433-08fd5bb5-168d-4059-966a-0537f1073fb8.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/217812578-f14c0c9b-c0ba-4889-92d0-c6955dbdfe98.png)